ntbs-service/Pages/Shared/_NotificationLayoutWithSideMenu.cshtml

```diff
@@ -16,6 +16,16 @@
     var endClass = $"notification-page-end notification-page-end{classSuffix}";
 }
 
+<nav class="nhsuk-breadcrumb" aria-label="Breadcrumb">
+  <div class="nhsuk-width-container">
+    <ol class="nhsuk-breadcrumb__list">
+      <li class="nhsuk-breadcrumb__item"><a class="nhsuk-breadcrumb__link" href="@Model.PreviousPageUrl"> @Model.PreviousPageLabel </a></li>
+      <li class="nhsuk-breadcrumb__item"><a class="nhsuk-breadcrumb__link" href="@Model.NotificationUrl"> Notifications </a></li>
+    </ol>
+  </div>
+</nav>
```

Comment:
Could we not use the taghelpers here? https://github.com/nhsuk/frontend-dotnetcore/tree/master/src/NHSUKFrontEndLibraryTagHelpers/NHSUK.FrontEndLibrary.TagHelpers/Tags/BreadCrumb#taghelper-markup

Reply:
Additionally, do we want to make a generalised partial out of this? If we do, it might make more sense to keep the breadcrumb items data in the `ViewBag` (or `ViewData`) rather than on the model - it seems a bit more natural to do that anyways, given that it's display-only context data!